admin/class-purger.php

```diff
@@ -226,61 +334,82 @@ private function _purge_by_options( $_post_ID, $blog_id, $_purge_page, $_purge_a
 		}
 	}
 
-	/*
+	/**
 	 * Deletes local cache files without a 3rd party nginx module.
 	 * Does not require any other modules. Requires that the cache be stored on the same server as WordPress. You must also be using the default nginx cache options (levels=1:2) and (fastcgi_cache_key "$scheme$request_method$host$request_uri").
 	 * Read more on how this works here:
 	 * https://www.digitalocean.com/community/tutorials/how-to-setup-fastcgi-caching-with-nginx-on-your-vps#purging-the-cache
+	 *
+	 * @param string $url URL to purge.
+	 *
+	 * @return bool
 	 */
-	private function _delete_cache_file_for( $url ) {
+	public function _delete_cache_file_for( $url ) {
 
-		// Verify cache path is set
+		// Verify cache path is set.
 		if ( ! defined( 'RT_WP_NGINX_HELPER_CACHE_PATH' ) ) {
+
 			$this->log( 'Error purging because RT_WP_NGINX_HELPER_CACHE_PATH was not defined. URL: ' . $url, 'ERROR' );
 			return false;
+
 		}
 
-		// Verify URL is valid
-		$url_data = parse_url( $url );
+		// Verify URL is valid.
+		$url_data = wp_parse_url( $url );
 		if ( ! $url_data ) {
+
 			$this->log( 'Error purging because specified URL did not appear to be valid. URL: ' . $url, 'ERROR' );
 			return false;
+
 		}
 
-		// Build a hash of the URL
+		// Build a hash of the URL.
 		$hash = md5( $url_data['scheme'] . 'GET' . $url_data['host'] . $url_data['path'] );
 
-		// Ensure trailing slash
+		// Ensure trailing slash.
 		$cache_path = RT_WP_NGINX_HELPER_CACHE_PATH;
-		$cache_path = ( '/' == substr( $cache_path, -1 ) ) ? $cache_path : $cache_path . '/';
+		$cache_path = ( '/' === substr( $cache_path, -1 ) ) ? $cache_path : $cache_path . '/';
 
-		// Set path to cached file
+		// Set path to cached file.
 		$cached_file = $cache_path . substr( $hash, -1 ) . '/' . substr( $hash, -3, 2 ) . '/' . $hash;
 
-		// Verify cached file exists
+		// Verify cached file exists.
 		if ( ! file_exists( $cached_file ) ) {
+
 			$this->log( '- - ' . $url . ' is currently not cached ( checked for file: ' . $cached_file . ' )' );
 			return false;
+
 		}
 
-		// Delete the cached file
+		// Delete the cached file.
 		if ( unlink( $cached_file ) ) {
 			$this->log( '- - ' . $url . ' *** PURGED ***' );
 		} else {
 			$this->log( '- - An error occurred deleting the cache file. Check the server logs for a PHP warning.', 'ERROR' );
 		}
+
 	}
 
-	private function _do_remote_get( $url ) {
+	/**
+	 * Remote get data from url.
+	 *
+	 * @param string $url URL to do remote request.
+	 */
+	public function _do_remote_get( $url ) {
```

Comment:
As I've mention earlier you should not use `_` prefix for public functions its meant to be use for private functions. 
Though this is old school from php4 coding standard but if we are following that use it only for `private` or `protected` methods or just remove underscore `_`.
ref: https://pear.php.net/manual/en/standards.naming.php
https://stackoverflow.com/questions/663350/whats-the-deal-with-a-leading-underscore-in-php-class-methods